features/proposed/FxA-70-kpi-dashboards/README.md

```diff
@@ -0,0 +1,414 @@
+
+2016 KPI Dashboards
+===================
+
+https://mozilla.aha.io/features/FXA-70
+
+## Stories
+
+As a product stakeholder for Firefox Accounts
+I want to quickly see the key metrics we care about for the year
+and how they're changing over time.
+
+## Overview
+
+The top-line objective for FxA in 2016 is to
+*seamlessly connect users to a compelling Firefox experience
+on all their devices*.
+To support this we will build a KPI dashboard
+that focusses on three key metrics:
+
+* Engagement Ratio,
+  as a measure of whether
+  they find the experience compelling.
+* Multi-Device Usage,
+  as a measure of how well we are
+  being present on all the user's devices.
+* Time taken for device connection,
+  as a measure of how seamlessly
+  we are making the connection.
+
+By default,
+each of these measures will be presented
+as an aggregation over all users.
+Linked detail dashboards will display them
+broken out by key user segmentation characteristics
+such as browser version,
+os platform,
+and whether they are a multi-device user.
+
+### Engagement Ratio
+
+This is the ratio of daily active users
+as a percentage of monthly active users,
+and is a measure of whether we have
+a compelling Firefox Experience.
+It does not meaningfully represent ongoing engagement
+with Firefox Accounts itself,
+since we're largely a background service
+once the user has connected their devices.
+
+To reflect this dependence on broader Firefox engagement levels,
+our target for 2016 will be twofold:
+
+* Increase engagement ratio by 10 percentage points over 2015 baseline.
```

Comment:
`over 2015 baseline.` do we have a graph or that data stored somewhere? Do we need a graph that is comparing this or will the dashboard just have it in the legend somewhere? 

`Increase engagement ratio by 10 percentage` I would set 2 goals 5% and 10% here because this is the first year we are doing this. Also spending time building the metrics dashboards instead of going full power on product development. (just my 2 cents though :wink: )